koku/api/settings/tag_management.py

```diff
@@ -0,0 +1,222 @@
+#
+# Copyright 2020 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+"""Data Driven Component Generation for Tag Management Settings."""
+from django.test import RequestFactory
+from rest_framework.serializers import ValidationError
+from tenant_schemas.utils import schema_context
+
+from api.common import error_obj
+from api.provider.models import Provider
+from api.query_params import QueryParameters
+from api.settings.utils import create_dual_list_select
+from api.settings.utils import create_plain_text
+from api.settings.utils import create_plain_text_with_doc
+from api.settings.utils import create_subform
+from api.settings.utils import generate_doc_link
+from api.settings.utils import SETTINGS_PREFIX
+from api.tags.aws.queries import AWSTagQueryHandler
+from api.tags.aws.view import AWSTagView
+from api.tags.azure.queries import AzureTagQueryHandler
+from api.tags.azure.view import AzureTagView
+from api.tags.ocp.queries import OCPTagQueryHandler
+from api.tags.ocp.view import OCPTagView
+from koku.cache import invalidate_view_cache_for_tenant_and_source_type
+from reporting.models import AWSEnabledTagKeys
+from reporting.models import AzureEnabledTagKeys
+from reporting.models import OCPEnabledTagKeys
+
+
+obtainTagKeysProvidersParams = {
```

Comment:
Lets wrap this in:
```
from django.conf import settings
if settings.DEVELOPMENT:
  openshift, aws, azure
else:
  openshift
```
We can bring AWS and Azure out as we finish the backend processing work.